metaci/release/utils.py

```diff
@@ -185,17 +191,24 @@ def send_start_webhook(release, role, config_item):
     }
     token = jwt_for_webhook()
     response = requests.post(
-        f"{settings.METACI_RELEASE_WEBHOOK_URL}/implementation_step_id/{implementation_step_id}/start/",
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/implementation/{implementation_step_id}/start/",
         json=payload,
         headers={"Authorization": f"Bearer {token}"},
     )
     result = response.json()
-    if result["success"]:
+    if (
+        "results" in result
+        and len(result["results"]) > 0
+        and result["results"][0]["success"]
+    ):
         logger.info(
             f"Successfully started implementation_step: {implementation_step_id}"
         )
+        return
     else:
-        raise Exception("\n".join(err["message"] for err in result["errors"]))
+        raise Exception(
+            "\n".join(err["message"] for err in result["results"]["errors"])
```

Comment:
This is not the format returned by gus-bus if there was an error: https://github.com/SFDO-Tooling/gus-bus/blob/main/gus_bus/main.py#L99